packages/eslint-plugin-svelte/src/rules/no-dupe-on-directives.ts

```diff
@@ -15,7 +15,16 @@ export default createRule('no-dupe-on-directives', {
 			duplication:
 				'This `on:{{type}}` directive is the same and duplicate directives in L{{lineNo}}.'
 		},
-		type: 'problem'
+		type: 'problem',
+		conditions: [
+			{
+				svelteVersions: ['3/4']
+			},
+			{
+				svelteVersions: ['5'],
+				runes: [false, 'undetermined']
```

Comment:
`on:` is deprecated but I think it can still be used, so I think we need the rule to work in rune mode too.